### ISSUE TITLE
`xargs` usage in demo script does not work on macOS

### ISSUE THREAD

**Issue by TobiasKadelka** [psyinfra/onyo#591]
```
❱ onyo get --match type=laptop -H --keys path | xargs -d "\n" onyo -y set --keys USB_A=2 --path

xargs: invalid option -- d
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements] [-S replsize]]
             [-J replstr] [-L number] [-n number [-x]] [-P maxprocs]
             [-s size] [utility [argument ...]]
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by TobiasKadelka** [comment]
Notes:
1. Currently working on both systems would be:
   `onyo set --keys USB_A=2 --path $(onyo get --match type=laptop -H --keys path)`
2. In the future this solution will also work (a.k.a. like above, but without xargs) 
   `onyo get --match type=laptop -H --keys path | onyo set --keys USB_A=2`

Once functioning and tested both variants should be described under https://onyo.readthedocs.io/en/latest/

**Comment by aqw** [comment]
The fix was merged in #602

Closing. :-)

**Comment by TobiasKadelka** [comment]
My plan was to first verify if it now actually does work on macOS or something else would happen :P
And yes, works just like intended.
